src/main/resources/META-INF/mods.toml

```diff
@@ -53,11 +53,5 @@ MineColonies is a colony simulator within Minecraft! There are numerous types of
     versionRange="[1.16.3, 1.17)"
     ordering="NONE"
     side="BOTH"
-# Here's another dependency
-[[dependencies.minecolonies]]
-    modId="structurize"
-    mandatory=true
-    versionRange="[0.13.177-ALPHA]"
-    ordering="AFTER"
-    side="BOTH"
+
```

Comment:
...Structurize is no longer a dep? What?

Reply:
Debug fail add =D

Reply:
yep, fixed :D

Reply:
raven pls stop auto merging the newest, its annoying!